mpc-core/src/point_addition/mod.rs

```diff
@@ -5,50 +5,51 @@
 //! would also be possible using this approach, but it hasn't been
 //! implemented here).
```

Comment:
Can we also leave a link here to the docs where this protocol can be referenced?